quinn-proto/src/connection/streams/state.rs

```diff
@@ -627,7 +643,7 @@ impl StreamsState {
         for dir in Dir::iter() {
             for index in 0..self.next[dir as usize] {
                 let id = StreamId::new(Side::Client, dir, index);
-                let stream = self.send.get_mut(&id).unwrap();
+                let stream = self.send.get_mut(&id).and_then(|s| s.as_mut()).unwrap();
```

Comment:
It's non-obvious that we'll never have `None`s in the map here. For robustness, let's handle those by `continue`ing rather than panicking, in the spirit of the branch below, since unallocated streams can't have outgoing data.